src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1738,8 +1745,9 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
 
         $sourceRecordID = $this->sId;
 
-        if (!$bIsUpdateCall) {
-            $bWasInserted = false;
+        if (false === $bIsUpdateCall) {
+            // handle insert query
```

Comment:
With the changed variable name (is update call) these two comments are not additionally helpful anymore.

Reply:
removed the "handle insert query" comment and the "handle update query" comment